### ISSUE TITLE
Diamondback only grants 2 revenge crits

### ISSUE THREAD

**Issue by Starblaster64** [WildCard65/Versus-Saxton-Hale#25]
After an update a few months back, Valve changed the Diamondback to only grant its backstab revenge crit on a backstab that actually killed someone. As a result, you only get 2 revenge crits per stab instead of 3.

**Comment by Chdata** [comment]
Yes, I was planning to change diamondback to 2 revenge crits. Not because of Valve though, for balancing. I'm also lowering the damage to 200. It really shouldn't be as good as amby..... well better.
